pkg/network/ovn_kubernetes_test.go

```diff
@@ -669,11 +677,37 @@ func TestOVNKubernetesIsSafe(t *testing.T) {
 
 	next.Migration.MTU.Network.From = prev.DefaultNetwork.OVNKubernetesConfig.MTU
 
+	// invalid Migration.MTU.Host.To, lower than minimum MTU for IPv4
```

Comment:
Shouldn't the comment be: `// invalid Migration.MTU.Network.To, lower than minimum MTU for IPv4`